OloEngine/src/OloEngine/Renderer/QualityTiering.cpp

```
@@ -0,0 +1,193 @@
+#include "OloEnginePCH.h"
+#include "QualityTiering.h"
+
+namespace OloEngine
+{
+    QualityTieringSettings GetPresetSettings(QualityPreset preset)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)